reference/query-hours.mdx

```diff
@@ -19,6 +18,18 @@ Axiom uses serverless computing to execute queries efficiently. The consumption
 - Time: The duration (in milliseconds) for which the serverless function is running to execute your query.
 - Memory allocation: The amount of memory (in GB) allocated to the serverless function during execution.
 
+## What counts as a query
+
+In calculating query costs, Axiom considers any request that queries your data as a query. For example, the following all count as queries:
+
+- You initiate a query in the Axiom user interface.
+- You query your data with an API token or a personal access token.
+- Your match monitor runs a query to determine if any new events match your criteria.
```

Comment:
it's not just  a match monitor, it's any type of monitor

Reply:
@ronoc Thanks! Yes, that's true. This list isn't meant to be exhaustive. It just gives a list of possible queries. I would leave it as it is.